Readme.md

```diff
@@ -1,28 +1,102 @@
-# Awesome Books Project
+<!-- <a name="readme-top"></a> -->
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- <div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome Books ](#-awesome-books-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [Getting Started ](#getting-started-)
+    - [Install ](#install-)
+  - [Authors ](#authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [Show your support ](#show-your-support-)
+  - [Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books <a name="about-project"></a>
 
 > This is a simple project that allow the user to added and/or remove books from a list. The data is persisted in the local storage.
 
-## Built With
+## 🛠 Built With <a name="built-with"></a>
 
 - Html
 - Css
-- JavaScript
+- Javascript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **Ability to add and remove books**
+- **Books saved in local storage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Live Demo (if available)
+<!-- LIVE DEMO -->
 
-[Live Demo Link](https://serengia.github.io/awesome-books-project/)
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## Getting Started
+> You can see a live demo via the link bellow.
+
+- [Live Demo Link](https://serengia.github.io/awesome-books-project)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running follow these simple example steps.
 
-### Install
+### Install <a name="install"></a>
 
 ```js
 npm install
 ```
 
-## Authors
+## Authors <a name="authors"></a>
 
 👤 **Author1**
 
```

Comment:
- [ ] [OPTIONAL] It would be a best practice` to enter the correct information in `Author 1` as in `Author 2` 😃